openedx/core/djangoapps/site_configuration/middleware.py

```diff
@@ -53,3 +54,26 @@ def _set_cookie_wrapper(key, value='', max_age=None, expires=None, path='/', dom
             response.set_cookie = _set_cookie_wrapper
 
         return response
+
+
+class AuthorizationCheckMiddleware(object):
+    """
+    Middleware for checking the authorization of users.
+    if not authorized and  site's setting 'DISABLE_CHECK_AUTHORIZATION' is False
```

Comment:
New sentence should start from the capital character.

Reply:
I fixed that mistake. Thank's